sim/death_knight/runeforging.go

```diff
@@ -100,7 +101,7 @@ func init() {
 			},
 		})
 
-		character.ItemSwap.RegisterOnSwapItemForEffectWithPPMManager(3368, 2.0, aura.Ppmm, aura)
+		character.ItemSwap.RegisterPPMEffect(3368, 2.0, aura.Ppmm, aura)
```

Comment:
Same question as for Tailoring enchants: are Runeforges fixed across item swaps? Because my understanding is that `RegisterPPMEffect()` will adjust the proc chance dynamically when weapons are swapped, but it won't actually disable the proc if the new weapon uses a different Runeforge.

Reply:
I added some changes which should now actually enable/disable aura's on the same item when they cannot be found on the equipped item.